Assets/_Game/Scripts/Player/PlayerController.cs

```diff
@@ -0,0 +1,80 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerController : MonoBehaviour
+{
+    public int health = 10;
+    public float speed;
+    public float attackSpeed = 0.5f;
+    public int damage;
+    public InventoryObject inventory;
+
+    private Vector2 movement;
+    private Vector2 lastMove;
+    [SerializeField]
+    private Animator playerAnim;
+    private void OnEnable()
+    {
+        EventManager.onItemUse += EventManager_onItemUse;

```

Comment:
Suggestion: Naming your event methods like this makes it really hard to read. It's very easy to confuse the local method with the actual `EventManager.onItemUse` event itself.